### PR TITLE
docs: linked to Explore Apps in relevant places

### PR DESCRIPTION
**Pull request by matryer**
**What is this feature?**

I have linked to Explore Apps (the simplified exploration docs) in a few relevant places to assist with discovery.

**Why do we need this feature?**

We should make people aware of Explore Apps as an option early in their learning process.

**Who is this feature for?**

This is for non-expert SREs who may want to skip learning query languages in favour of a point-and-click experience.

### CODE REVIEW

**Review by jdbaldry — COMMENTED** [review]
I'll let a technical writer do a full review but we will want to remove the use of "new" because we want to keep documentation timeless: https://developers.google.com/style/timeless-documentation

**Comment by matryer** [comment]
thanks @jdbaldry - I've updated the PR

File: docs/sources/explore/_index.md [hunk 1]
```diff
@@ -57,6 +60,10 @@ cards:
 
 Explore is your starting point for querying, analyzing, and aggregating data in Grafana. You can quickly begin creating queries to start analyzing data without having to create a dashboard or customize a visualization.
 
+{{% admonition type="note" %}}
+If you're using Mimir, Loki, Tempo or Pyroscope, be sure to try our [simplified queryless Explore experiences]({{< relref "simplified-exploration" >}}).
```
Comment (knylander-grafana) [inline docs/sources/explore/_index.md:64]: In the technical docs, we try to [minimize "we" and "our"](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#guidelines-for-whats-new-content): "Our style normally prefers second person (“you”) and avoids first person (“I”/“we”)."
```suggestion
If you're using Mimir, Loki, Tempo, or Pyroscope, try the [simplified queryless Explore experiences]({{< relref "simplified-exploration" >}}).
```

File: docs/sources/introduction/_index.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ After you have [installed Grafana]({{< relref "../setup-grafana/installation" >}
 
 The following sections provide an overview of Grafana features and links to product documentation to help you learn more. For more guidance and ideas, check out our [Grafana Community forums](https://community.grafana.com/).
 
-## Explore metrics, logs, and traces
+## Explore metrics, logs, traces and profiles
```
Comment (knylander-grafana) [inline docs/sources/introduction/_index.md:23]: Each of the Explore apps is a product/feature name, which means they should be proper nouns. i'm not sure if we can use proper nouns in a list like this. I'll ask the other writers for input.

File: docs/sources/shared/basics/what-is-grafana.md [hunk 3]
```diff
@@ -22,6 +22,10 @@ Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing
 
 Grafana Mimir is an open source software project that provides a scalable long-term storage for Prometheus. For more information about Grafana Mimir, refer to [Grafana Mimir documentation](/docs/mimir/latest/).
 
-### Grafana Oncall
+### Grafana OnCall
 
 Grafana OnCall is an open source incident response management tool built to help teams improve their collaboration and resolve incidents faster. For more information about Grafana OnCall, refer to [Grafana OnCall documentation](/docs/oncall/latest/).
+
+### Explore Apps
```
Comment (knylander-grafana) [inline docs/sources/shared/basics/what-is-grafana.md:29]:
```suggestion
### Explore apps suite
```

File: docs/sources/shared/basics/what-is-grafana.md [hunk 4]
```diff
@@ -22,6 +22,10 @@ Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing
...
+
+Our new suite of apps provides queryless point-and-click experiences for working with Mimir, Loki, Tempo and Pyroscope data. Diagnose issues and understand your systems faster than ever regardless of your level of expertise. For more information, check out [Explore Apps simplified exploration documentation](/docs/grafana/latest/explore/simplified-exploration/).
```
Comment (knylander-grafana) [inline docs/sources/shared/basics/what-is-grafana.md:31]:
```suggestion
The new new suite of Explore apps provides queryless point-and-click experiences for working with Mimir, Loki, Tempo, and Pyroscope data.
You can diagnose issues and understand your systems faster regardless of your level of expertise.
For more information, refer to the [Explore apps documentation](/docs/grafana/latest/explore/simplified-exploration/).
```

File: docs/sources/datasources/loki/_index.md [hunk 5]
```diff
@@ -158,3 +158,7 @@ Grafana lists these variables in dropdown select boxes at the top of the dashboa
 Grafana refers to such variables as template variables.
 
 For details, see the [template variables documentation]({{< relref "./template-variables" >}}).
+
+## Try Explore Logs
+
+Try our [simplified queryless Explore experiences]({{< relref "../../explore/simplified-exploration/logs" >}}) for Loki.
```
Comment (knylander-grafana) [inline docs/sources/datasources/loki/_index.md:164]:
```suggestion
Try [Explore Logs]({{< relref "../../explore/simplified-exploration/logs" >}}) for a simplified, queryless Explore experience.
```

File: docs/sources/introduction/_index.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/sources/introduction/_index.md:23]:
```suggestion
## Explore metrics, logs, traces, and profiles
```

File: docs/sources/introduction/_index.md [hunk 6]
```diff
@@ -20,9 +20,9 @@ After you have [installed Grafana]({{< relref "../setup-grafana/installation" >}
...
 
-Explore your data through ad-hoc queries and dynamic drilldown. Split view and compare different time ranges, queries and data sources side by side. Refer to [Explore]({{< relref "../explore" >}}) for more information.
+Explore your data through ad-hoc queries and dynamic drilldown. Split view and compare different time ranges, queries and data sources side by side. Refer to [Explore]({{< relref "../explore" >}}) for more information. Or use our new suite of [Explore Apps for simplified exploration]({{< relref "../explore/simplified-exploration" >}}) to browse your telemetry data without writing queries.
```
Comment (knylander-grafana) [inline docs/sources/introduction/_index.md:25]:
```suggestion
Explore your data through ad-hoc queries and dynamic drilldown.
Split view and compare different time ranges, queries and data sources side by side. 
Refer to [Explore]({{< relref "../explore" >}}) for more information. 

You can also use one of the [Explore Apps for simplified exploration]({{< relref "../explore/simplified-exploration" >}}) to browse your telemetry data without writing queries.
```

**Review by knylander-grafana — APPROVED** [review]
Thank you for updating the docs!

**Review by jdbaldry — COMMENTED** [review]
Note to the docs team, the following files need to use `ref` URIs (or relative paths where it works) because the content is reused in Grafana Cloud:

- `docs/sources/datasources/loki/_index.md`
- `docs/sources/explore/_index.md`

**Pull request closed without merging**